### DISCUSSION TITLE
PCAP Jobs are all "Incomplete"

### DISCUSSION THREAD

**Discussion by scout213** [Security-Onion-Solutions/securityonion#10574] [Unsupported Versions] [answered]
I have a fresh install of 2.3.250.  When I run so-status, all of the services appear to be running.  When I drill down into an alert, and then try to use the action menu to pull the PCAP, I get an "Incomplete" error every time.  I checked the pcap folder, and it's clearly filling with data.  The only non standard configuration are the steps outlined here: https://github.com/Security-Onion-Solutions/securityonion/discussions/2164 to store PCAP on spinning rust and use the SSDs for everything else.  Server specs are:

Dell R730
2 x E5-2697A v4 (2 x 16 Core)
256GB RAM
1TB Samsung 870 Evo (Boot SSD)
8TB Intel P4510 (/nsm storage)
8 x 14TB WD SAS in RAID 10 (/nsm/{pcap,pcapout,pcaptmp} storage)

I added the following line to fstab: 

/dev/sda                /mnt/pcap               xfs     defaults,noatime        0 0

I've attached a few screenshots to illustrate the workflow and troubleshooting steps I've taken.

<img width="714" alt="Screenshot 2023-06-12 at 22 40 34" src="https://github.com/Security-Onion-Solutions/securityonion/assets/22671045/a19734b8-47bc-4a7c-9f1d-111747d7ee22">

<img width="1464" alt="Screenshot 2023-06-12 at 22 38 15" src="https://github.com/Security-Onion-Solutions/securityonion/assets/22671045/b0a60f37-d895-461c-b3e0-b7de03be8970">

<img width="1463" alt="Screenshot 2023-06-12 at 22 38 31" src="https://github.com/Security-Onion-Solutions/securityonion/assets/22671045/d00c5f2b-7e26-4e84-9d40-09436fc0a2f2">

<img width="1464" alt="Screenshot 2023-06-12 at 22 38 41" src="https://github.com/Security-Onion-Solutions/securityonion/assets/22671045/1d3e95a8-7c66-4558-96a1-3107460e2f29">

This is the output of ls -alh for the directory /nsm/pcap/

<img width="712" alt="Screenshot 2023-06-12 at 22 39 49" src="https://github.com/Security-Onion-Solutions/securityonion/assets/22671045/7be3c860-fc68-4d21-9211-97ecac590ea1">

**Comment by dougburks** [comment, marked as answer]
I'd start by checking the log files in `/opt/so/log/`. 

If that doesn't help, then you might want to install a fresh copy of Security Onion on a separate machine (perhaps in a VM) and compare the systems side-by-side looking for differences in file permissions, ownership, SELinux context, etc.
